src/daikinapi.ts

```diff
@@ -264,27 +273,32 @@ export class DaikinApi {
   private async getToken() {
     this.log.debug('Getting token...');
     try {
-      const response = await axios.post(
-        'https://api.daikinskyport.com/users/auth/login',
-        {
+      const response = await fetch(DAIKIN_API_LOGIN_URL, {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
           email: this.user,
           password: this.password,
-        },
-        {
-          headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/json',
-          },
-        },
-      );
-      this.setToken(response);
+        }),
+      });
+
+      if (!response.ok) {
+        const errorMessage = JSON.stringify(response.json());
+        throw new Error(errorMessage);
```

Comment:
```suggestion
        throw new Error(`Request ${DAIKIN_API_LOGIN_URL} failed with status ${response.status} - ${response.statusText}:\n${response.text()}`);
```
might work better here since the status is most likely quite informative and the body might or might not be json in the error case and text() should display it either way.

Reply:
applies to the other catch blocks below as well